sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/autoconfigure/aad/AADAuthenticationProperties.java

```diff
@@ -299,4 +311,22 @@ public boolean isAllowedGroup(String group) {
                        .orElseGet(Collections::emptyList)
                        .contains(group);
     }
+
+    @Override
+    public void afterPropertiesSet() throws Exception {
+        if (!StringUtils.hasText(graphMembershipUri)) {
+            graphMembershipUri = graphBaseUri + "v1.0/me/memberOf";
+        }
+        validate();
+    }
+
+    private String addSlash(String uri) {
+        return uri.endsWith("/") ? uri : uri + "/";
+    }
+
+    private void validate() {
```

Comment:
Rename to `validateGraphUris` will make the purpose clearer.
And I prefer to delete this method, move the 3 lines of code into `afterPropertiesSet`.